codespell_lib/data/dictionary.txt

```diff
@@ -567,9 +567,10 @@ acquited->acquitted
 acqure->acquire
 acqured->acquired
 acqures->acquires
+acquries->acquires
 acquring->acquiring
 acrage->acreage
-acrued->accured
+acrued->accrued, accured,
```

Comment:
Is accured a word, I can only find accrued and accursed.

Reply:
I'm not sure. First I wanted to throw it out too, because it is not in aspell.
Then, I thought, I found it in an other dictionary. But I cannot find it right now.

I added it as mistake.

Reply:
Do you want to take a bit more time to double-check? Given we're correcting mistakes it would be worth trying to avoid making any more. :laughing:

Reply:
I didn't add accured :-D

Reply:
and I only wanted to remove stuff, where I was sure, that it is wrong.